### ISSUE TITLE
Simple cyst

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5030]
Definition:Epithelium lined space (squamous/columnar) without septations

**Comment by pnrobinson** [comment]
@avizrosenberg
@yongqunh
@beckysteck

Can you please check the proposed revision to the term name and my definition?

Label: Simple renal cyst
Synonyms: Simple kidney cyst, Stage I renal cyst
Def:  A solitary and unilateral structure located in the renal cortex. Simple renal cysts have a smooth wall lined by a single layer of epithelial cells. The typical ultrasound criteria for defining a simple renal cyst include round/oval shape; thin, smooth walls; posterior enhancement; no internal debris or septa.
Comment: Simple renal cysts are usually asymptomatic.
PMID:25165175
SEE https://www.ncbi.nlm.nih.gov/pubmed/25165175/

**Comment by mellybelly** [comment]
probably need to align with NCIT; https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C3970
there are quite a few non-malignant pathological anatomical entities in NCIT, and we don't really have a home for them in the OBO Library.
@cmungall

**Comment by pnrobinson** [comment]
Please see also this issue: https://github.com/obophenotype/human-phenotype-ontology/issues/5031
The cited article says that sonographers/rediologists have agreed on 5 stages -- thus, we may want more than simple/complex. I cannot find this in NCIT, and if the nephrologists think it is a good idea, we should also feed back to NCIT.

**Comment by cmungall** [comment]
We have a bunch of cysts in mondo, mapped to NCIT, should also be
non-equivalence mapped to the overlapping cysts in HP. Seem more phenotype
than disease

**Comment by pnrobinson** [comment]
leave off imaging criteria

**Comment by pnrobinson** [comment]
@avizrosenberg
I see now that there is a term "Solitary renal cyst HP:0012581" (with synonym: "Simple renal cyst"). 
https://hpo.jax.org/app/browse/term/HP:0012581

I would change the primary label to "Simple renal cyst" -- is that OK?

**Comment by pnrobinson** [comment]
I will do so because Simple kindney cyst was already listed as a synonym
